css-pseudo-classes/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Pseudo Classes</title>
+</head>
+<body>
+  <h1>Hover Button</h1>
+  <div>
+    <button>I'm a button!</button>
+  </div>
+  <h1>Striped Table</h1>
```

Comment:
We generally want to avoid having more than one `h1` tag on one page. This is something that I missed so sorry about that. It's for more of SEO practice.